pkg/agent/baker_test.go

```diff
@@ -1580,41 +1192,6 @@ testdomain567.com:53 {
 				config.DisableUnattendedUpgrades = false
 			}, nil),
 
-		Entry("AKSUbuntu1804 with containerd and kubenet cni", "AKSUbuntu1804+Containerd+Kubenet+FIPSEnabled", "1.19.13",
```

Comment:
we need to make sure we're not losing base-level test coverage, this might need to be `AKSUbuntu2204 with containerd and kubenet cni` if that test case doesn't already exist - same comment applies to all test cases you're removing here

Reply:
right, I will ensure all 1604/1804 test are called out explicitely with the SKU version and ensure we are not loosing coverage